test/addons/rook-operator/kustomization.yaml

```diff
@@ -4,41 +4,42 @@
 # yamllint disable rule:line-length
 ---
 resources:
-  - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/crds.yaml
-  - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/common.yaml
-  - https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/operator.yaml
-patchesJson6902:
-  - target:
-      kind: ConfigMap
-      name: rook-ceph-operator-config
-      namespace: rook-ceph
-    patch: |-
-      - op: add
-        path: /data/CSI_ENABLE_CSIADDONS
-        value: 'true'
-      - op: add
-        path: /data/CSI_ENABLE_OMAP_GENERATOR
-        value: 'true'
+- https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/crds.yaml
+- https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/common.yaml
+- https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/operator.yaml
 
+# Disable to avoid random failures when restaring the enviroment.
+# failed to call webhook:
+# Post "https://rook-ceph-admission-controller.rook-ceph.svc:443/
+# validate-ceph-rook-io-v1-cephblockpool?timeout=5s":
+# x509: certificate signed by unknown authority
 patches:
-  # Disable to avoid random failures when restaring the enviroment.
-  #   failed to call webhook:
-  #   Post "https://rook-ceph-admission-controller.rook-ceph.svc:443/
-  #   validate-ceph-rook-io-v1-cephblockpool?timeout=5s":
-  #   x509: certificate signed by unknown authority
-  - target:
-      kind: Deployment
-      name: rook-ceph-operator
-    patch: |-
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: not-used
-      spec:
-        template:
-          spec:
-            containers:
-              - name: rook-ceph-operator
-                env:
-                  - name: ROOK_DISABLE_ADMISSION_CONTROLLER
-                    value: 'true'
+- patch: |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: not-used
+    spec:
+      template:
+        spec:
+          containers:
+            - name: rook-ceph-operator
+              env:
+                - name: ROOK_DISABLE_ADMISSION_CONTROLLER
+                  value: 'true'
+  target:
+    kind: Deployment
+    name: rook-ceph-operator
+- patch: |-
+    - op: add
+      path: /data/CSI_ENABLE_CSIADDONS
+      value: 'true'
+    - op: add
+      path: /data/CSI_ENABLE_OMAP_GENERATOR
+      value: 'true'
+  target:
+    kind: ConfigMap
+    name: rook-ceph-operator-config
+    namespace: rook-ceph
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Same as previous file, keep the original indentation and move apiVersion and kind to the top.